### ISSUE TITLE
Instructions say, type 'xonda' but 'command not found'

### ISSUE THREAD

**Issue by michaelaye** [gforsyth/xonda#37]
The README states:

> Then, use as you would use conda, but just write xonda instead.

But when I try that I get `command not found` ?

```bash
(py37) klay6683@MacL2938 ~/src/xonda master $ xonda list
xonsh: For full traceback set: $XONSH_SHOW_TRACEBACK = True
xonsh: subprocess mode: command not found: xonda
Did you mean one of the following?
    conda:  Alias
    xonsh:  Command (/Users/klay6683/miniconda3/envs/py37/bin/xonsh)
    xindy:  Command (/Library/TeX/texbin/xindy)
```

Trying this on macOS 10.13.6

**Comment by gforsyth** [comment]
Yep, sorry about that.  That particular instruction is out of date (and wrong, as you've noticed) -- when you load the `xonda` xontrib it will add an alias over `conda`

**Comment by gforsyth** [comment]
Fixed in #38 (updated instructions to remove outdated stuff)

**Comment by gforsyth** [comment]
Thanks for reporting this @michaelaye !
